CppSamples/Geometry/GeodesicOperations/GeodesicOperations.cpp

```diff
@@ -104,15 +106,17 @@ void GeodesicOperations::componentComplete()
 
     // densify the path as a geodesic curve and show it with the path graphic
     constexpr double maxSegmentLength = 1.0;
-    const LinearUnit unitOfMeasurement(LinearUnitId::Kilometers);
+    const LinearUnit linearUnitOfMeasurement(LinearUnitId::Kilometers);
+    const AngularUnit angularUnitOfMeasurement(AngularUnit::radians());
```

Comment:
There's a helper for this one as well, but your code is also correct. If we have helpers, it's easier to use those IMO.
```c++
const AngularUnit angularUnitOfMeasurementRads = AngularUnit::radians();
```
I snuck in the units in the var name as well.